hieradata/node/auxtel-dc01.tu.lsst.org.yaml

```diff
@@ -0,0 +1,28 @@
+---
+nm::connections:
+  ens2f0:
+    content:
+      connection:
+        id: "ens2f0"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "ens2f0"
+      ethernet: {}
+      ipv4:
+        method: "auto"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  ens2f1:
```

Comment:
There needs to be an interface named `lsst-daq` as this is used in the ccs/daq configuration.